### PR TITLE
[WEB-5549]: update billing address of the active subscriptions

### PR DESCRIPTION
**Pull request by aosmakserato**

### CODE REVIEW

File: src/Ecom/Command/UserBillingAddressUpdate.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            ],
+            'city' => [
+                'type'     => self::ARG_TYPE_STRING,
+                'required' => false
+            ],
+            'zip' => [
+                'type'     => self::ARG_TYPE_STRING,
+                'required' => false
+            ],
+            'country_code' => [
```
Comment (Jasmin12) [inline src/Ecom/Command/UserBillingAddressUpdate.php:68]: Isn't country compulsory?

Comment (aosmakserato) [reply src/Ecom/Command/UserBillingAddressUpdate.php:68]: No, it is not. That's an update, not create action. You don't have to pass the country code if you don't want to update it.

**Review by Jasmin12 — APPROVED** [review]
Looks good @aosmakserato , nice one!

**Review by GuillaumeBrousse — APPROVED** [review]
Straight forward, Nice one!

**Pull request merged**